x-pack/plugins/snapshot_restore/public/application/components/repository_form/type_settings/s3_settings.tsx

```diff
@@ -368,7 +337,20 @@ export const S3Settings: React.FunctionComponent<Props> = ({
           fullWidth
           isInvalid={Boolean(hasErrors && settingErrors.bufferSize)}
           error={settingErrors.bufferSize}
-          helpText={textService.getSizeNotationHelpText()}
+          helpText={
+            <FormattedMessage
+              id="xpack.snapshotRestore.repositoryForm.typeS3.bufferSizeHelpText"
+              defaultMessage="Accepts byte size units, for example: {example1}, {example2}, {example3}, {example4}. Defaults to the minimum between {defaultSize} and {defaultPercentage} of the heap size."
```

Comment:
The wording for the default wasn't clear to me. The docs aren't clear either so I'll get those fixed.
```suggestion
              defaultMessage="Accepts byte size units, such as {example1}, {example2}, {example3}, or {example4}. Defaults to {defaultSize} or {defaultPercentage} of JVM heap, whichever is smaller."
```